samples/opencl/FindOpenCL.cmake

```diff
@@ -0,0 +1,174 @@
+# Find OpenCL
```

Comment:
do you mean public khronos repo on GitHub? I did not find anything related on github.com/khronos